recipes/bliss/all/conanfile.py

```diff
@@ -23,73 +27,64 @@ class BlissConan(ConanFile):
         "fPIC": True,
         "with_exact_int": False,
     }
-    exports_sources = "CMakeLists.txt", "patches/**"
-    generators = "cmake"
 
-    _cmake = None
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    def export_sources(self):
+        export_conandata_patches(self)
 
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
     def configure(self):
         if self.options.shared:
-            del self.options.fPIC
+            self.options.rm_safe("fPIC")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
 
     def requirements(self):
         if self.options.with_exact_int == "gmp":
-            self.requires("gmp/6.2.0")
+            self.requires("gmp/6.2.1", transitive_headers=True, transitive_libs=True)
```

Comment:
Please stop exposing libs by default 

```suggestion
            self.requires("gmp/6.2.1", transitive_headers=True)
```

Reply:
Can you prove they are required? 

Moving forward the policy is going to be 

- if you dont know do not expose them
- if they are required put a comment

This way we can track and measure to test the hypothesis